i18n(zh-cn): Translation /features/clipboard.mdx

Yaxnice301 opened the pull request:
i18(zh-CN): Translation /features/clipboard.mdx

Yaxnice301 committed:
i18n(zh-cn): Translation /features/clipboard.mdx

Yaxnice301 commented:
Why did I add this page in the zh-cn directory, but it is not displayed in the menu bar? Did I forget to write some configuration?

Yaxnice301 committed:
i18(zh-CN): 翻译对话框页面为中文

Yaxnice301 committed:
i18n(zh-CN): 翻译os-info.mdx 为中文

Yaxnice301 committed:
change title

Yaxnice301 committed:
fix 修复对话框标题链接不对应的问题

vasfvitor commented on src/content/docs/zh-CN/features/os-info.mdx:
````diff
@@ -0,0 +1,105 @@
+```js
+import { platform } from '@tauri-apps/plugin-os';
+
+const platform = await platform();
````
could this wait until #1733 is in so lines 90, 91 are updated accordingly?

vasfvitor replied:
```suggestion
const currentPlatform = await platform();
```

vasfvitor commented on src/content/docs/zh-CN/features/os-info.mdx:
```diff
@@ -0,0 +1,105 @@
+import { platform } from '@tauri-apps/plugin-os';
+
+const platform = await platform();
+console.log(platform);
```
```suggestion
console.log(currentPlatform);
```

Yaxnice301 commented:
I don't understand, do you mean there will be an update to this document? If there is an update, just send me a message, and I will update the translation when I get the email.

vasfvitor commented:
>Why did I add this page in the zh-cn directory, but it is not displayed in the menu bar? Did I forget to write some configuration?

No no at all, I'm looking into it now and the issue is on the website. I'll open a pr to fix it soon.

> I don't understand, do you mean there will be an update to this document? If there is an update, just send me a message, and I will update the translation when I get the email.

exactly, I left a suggestion with the updated code, but you can look at the mentioned pr and make the changes yourself

thank you very much for this translations, I'm sorry for this trouble

Yaxnice301 commented:
#### I'm going to say something
- Thank you for your reply
- Nope

Yaxnice301 committed:
i18n(zh-cn)翻译 create a project

lorenzolewis committed:
Merge branch 'next' into next

lorenzolewis committed:
rename

lorenzolewis committed:
rename

lorenzolewis commented on src/content/docs/zh-cn/features/file-system.mdx:
This page is not ready to be translated because it is only a placeholder. Could you remove this file from the PR?

lorenzolewis commented on src/content/docs/zh-cn/guides/create/index.mdx:
This page is not ready to be translated because the content will be updated soon. Could you remove this file from the PR?

lorenzolewis reviewed (changes requested):
Thank you very much for this @Yaxnice301 ! I merged in #1747 and #1735 for the issue you raised (thanks @vasfvitor !)

I put a comment on two of the pages as they are not ready to be translated right now (but I really appreciate the eagerness to contribute!) You can check https://beta.tauri.app/contribute/translate-status to see which pages are ready for translation in the future.

@dklassic would you be able to help with a review please?

dklassic commented on src/content/docs/zh-cn/features/clipboard.mdx:
```diff
@@ -0,0 +1,120 @@
+    pnpm="pnpm tauri add clipboard-manager"
+    cargo="cargo tauri add clipboard-manager" />
+
+    2. 修改 `lib.rs` 以初始化插件:
```
```suggestion
    2. 修改 `lib.rs` 以初始化插件：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/clipboard.mdx:
```diff
@@ -0,0 +1,120 @@
+    </TabItem>
+    <TabItem label="Manual">
+
+    1. 运行 `cargo add tauri-plugin-clipboard-manager` 将插件添加到项目依赖 `Cargo.toml` 中.
```
```suggestion
    1. 运行 `cargo add tauri-plugin-clipboard-manager` 将插件添加到项目依赖 `Cargo.toml` 中。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/clipboard.mdx:
```diff
@@ -0,0 +1,120 @@
+
+    1. 运行 `cargo add tauri-plugin-clipboard-manager` 将插件添加到项目依赖 `Cargo.toml` 中.
+
+    2. 修改 `lib.rs` 以初始化插件:
```
```suggestion
    2. 修改 `lib.rs` 以初始化插件：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/clipboard.mdx:
````diff
@@ -0,0 +1,120 @@
+    }
+    ```
+
+    3. If you'd like to manage the clipboard in JavaScript then install the npm package as well:
````
missing translation

dklassic commented on src/content/docs/zh-cn/features/clipboard.mdx:
```diff
@@ -0,0 +1,120 @@
+
+{/* TODO: Link to which language to use, frontend vs. backend guide when it's made */}
+
+剪贴板插件在JavaScript和Rust中都可用。
```
```suggestion
剪贴板插件在 JavaScript 和 Rust 中都可用。
```
Formatting nitpick.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+    pnpm="pnpm tauri add dialog"
+    cargo="cargo tauri add dialog" />
+
+    2. 修改 `lib.rs` 以初始化插件:
```
```suggestion
    2. 修改 `lib.rs` 以初始化插件：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+    </TabItem>
+    <TabItem label="Manual">
+
+    1. 运行 `cargo add tauri-plugin-dialog` 将插件添加到项目依赖 `Cargo.toml` 中.
```
```suggestion
    1. 运行 `cargo add tauri-plugin-dialog` 将插件添加到项目依赖 `Cargo.toml` 中。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+    1. 运行 `cargo add tauri-plugin-dialog` 将插件添加到项目依赖 `Cargo.toml` 中.
+
+    2. 修改 `lib.rs` 以初始化插件:
```
```suggestion
    2. 修改 `lib.rs` 以初始化插件：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
````diff
@@ -0,0 +1,299 @@
+    }
+    ```
+
+    3. 如果您想在JavaScript中创建对话框，请同时安装npm包：
````
```suggestion
    3. 如果您想在 JavaScript 中创建对话框，请同时安装 npm 包：
```
Formatting nitpick.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+## 使用
+
+该对话框插件在JavaScript和Rust中都可用。以下是您可以使用它的方式：
```
```suggestion
该对话框插件在 JavaScript 和 Rust 中都可用。以下是您可以使用它的方式：
```
Formatting nitpick.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+该对话框插件在JavaScript和Rust中都可用。以下是您可以使用它的方式：
+
+使用 JavaScript:
```
```suggestion
使用 JavaScript：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+- [打开一个文件选择对话框](#打开一个文件选择对话框)
+- [保存文件对话框](#保存文件对话框)
+
+使用 Rust:
```
```suggestion
使用 Rust：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+### JavaScript
+
+查看所有 [对话框选项](/2/reference/js/dialog/) 在 JavaScript API 参考手册中。
```
```suggestion
查看所有[对话框选项](/2/reference/js/dialog/)在 JavaScript API 参考手册中。
```
Formatting nitpick.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+#### 创建消息对话框
+
+显示一个带有 `Ok` 按钮的对话框。 请注意：如何用户关闭对话框将返回 `false`.
```
```suggestion
显示一个带有 `Ok` 按钮的对话框。请注意：如何用户关闭对话框将返回 `false`。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+打开一个文件/文件夹选择对话框
+
+`multiple` 选项控制对话框是否允许多选,当 `directory` 决定是否为文件夹选择。
```
```suggestion
`multiple` 选项控制对话框是否允许多选，当 `directory` 决定是否为文件夹选择。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+#### 打开一个文件选择对话框
+
+打开一个文件/文件夹选择对话框
```
```suggestion
打开一个文件或文件夹选择对话框
```
Technically the same meaning but much better formatting simply using "或" here.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+#### 保存文件对话框
+
+打开一个 文件/文件夹 保存对话框.
```
```suggestion
打开一个文件或文件夹保存对话框.
```
Formatting correction and technically the same meaning but much better formatting simply using "或" here.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
````diff
@@ -0,0 +1,299 @@
+        .blocking_show();
+```
+
+如果您需要非阻塞操作，您可以改用 `show()`:
````
```suggestion
如果您需要非阻塞操作，您可以改用 `show()`：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+
+#### 构建消息对话框
+
+显示一个带有 `Ok` 按钮的对话框。 请注意：如何用户关闭对话框将返回 `false`.
```
```suggestion
显示一个带有 `Ok` 按钮的对话框。请注意：如何用户关闭对话框将返回 `false`。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
````diff
@@ -0,0 +1,299 @@
+    .blocking_show();
+```
+
+如果您需要非阻塞操作，您可以改用 `show()`:
````
```suggestion
如果您需要非阻塞操作，您可以改用 `show()`：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+use tauri_plugin_dialog::DialogExt;
+
+let file_path = app.dialog().file().blocking_pick_file();
+// 返回一个 文件路径 `Option`, 如何用户关闭了对话框则返回 `None`。
```
```suggestion
// 返回一个文件路径 `Option`，如果用户关闭了对话框则返回 `None`。
```
Typo fix and formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
````diff
@@ -0,0 +1,299 @@
+// 返回一个 文件路径 `Option`, 如何用户关闭了对话框则返回 `None`。
+```
+
+If you need a non blocking operation you can use `show()` instead:
````
Missing translation.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+use tauri_plugin_dialog::DialogExt;
+
+app.dialog().file().pick_file(|file_path| {
+    // 返回一个 文件路径 `Option`, 如何用户关闭了对话框则返回 `None`。
```
```suggestion
    // 返回一个文件路径 `Option`，如果用户关闭了对话框则返回 `None`。
```
Typo fix and formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
````diff
@@ -0,0 +1,299 @@
+    // 如果用户关闭了对话框 file path 为 `None`
+```
+
+代替方案:
````
```suggestion
代替方案：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+    .file()
+    .add_filter("My Filter", &["png", "jpeg"])
+    .pick_file(|file_path| {
+        // 返回一个 文件路径 `Option`, 如何用户关闭了对话框则返回 `None`。
```
```suggestion
        // 返回一个文件路径 `Option`，如果用户关闭了对话框则返回 `None`。
```
Typo fix and formatting correction.

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
Is there a particular reason why both `clipboard` and `dialog` plugins have full translated names but this one in particular only has a translated title but left untranslated in the article?

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
```diff
@@ -0,0 +1,105 @@
+    pnpm="pnpm tauri add os"
+    cargo="cargo tauri add os" />
+
+    2. 修改 `lib.rs` 以初始化插件:
```
```suggestion
    2. 修改 `lib.rs` 以初始化插件：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
```diff
@@ -0,0 +1,105 @@
+    </TabItem>
+    <TabItem label="Manual">
+
+    1. 运行 `cargo add tauri-plugin-os` 请插件添加到项目依赖 `Cargo.toml` 中.
```
```suggestion
    1. 运行 `cargo add tauri-plugin-os` 请插件添加到项目依赖 `Cargo.toml` 中。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
```diff
@@ -0,0 +1,105 @@
+
+    1. 运行 `cargo add tauri-plugin-os` 请插件添加到项目依赖 `Cargo.toml` 中.
+
+    2. 修改 `lib.rs` 以初始化插件:
```
```suggestion
    2. 修改 `lib.rs` 以初始化插件：
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
````diff
@@ -0,0 +1,105 @@
+    }
+    ```
+
+    3. 如果您想在JavaScript中使用，请同时安装npm包：
````
```suggestion
    3. 如果您想在 JavaScript 中使用，请同时安装 npm 包：
```
Formatting nitpick.

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
```diff
@@ -0,0 +1,105 @@
+
+## 使用
+
+使用此插件，您可以从当前操作系统查询多个信息。在 [JavaScript API](/2/reference/js/os/) 或 [Rust API](https://docs.rs/tauri-plugin-os/) 中查看所有可用函数.
```
```suggestion
使用此插件，您可以从当前操作系统查询多个信息。在 [JavaScript API](/2/reference/js/os/) 或 [Rust API](https://docs.rs/tauri-plugin-os/) 中查看所有可用函数。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
```diff
@@ -0,0 +1,105 @@
+
+#### 示例：操作系统平台
+
+`platform` 返回一个描述所使用操作系统的字符串。该值在编译时设置。可能的值是 `linux`, `macos`, `ios`, `freebsd`, `dragonfly`, `netbsd`, `openbsd`, `solaris`, `android`, `windows`.
```
```suggestion
`platform` 返回一个描述所使用操作系统的字符串。该值在编译时设置。可能的值是 `linux`、`macos`、`ios`、`freebsd`、`dragonfly`、`netbsd`、`openbsd`、`solaris`、`android`、`windows`。
```
Formatting correction.

dklassic commented on src/content/docs/zh-cn/features/os-info.mdx:
````diff
@@ -0,0 +1,105 @@
+```rust
+let platform = tauri_plugin_os::platform();
+println!("Platform: {}", platform);
+// Prints "windows" to the terminal
````
untranslated comment?

dklassic commented on src/content/docs/zh-cn/features/clipboard.mdx:
```diff
@@ -0,0 +1,120 @@
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialize the plugin
```
untranslated comment?

dklassic reviewed (changes requested):
Thanks for contributing! The translation is generally in good shape but there are some formatting issues and missing translations from place to place. One thing in particular stand out is that both `clipboard` and `dialog` plugins have full translated names while `OS Information` is translated in the title but left untouched within the article, is there a reason behind this?

simonhyll committed:
Merge branch 'v2' into next

simonhyll committed:
Merge branch 'v2' into next

simonhyll committed:
Merge branch 'v2' into next

renovate[bot] committed:
fix(deps): update dependency astro to v4.9.0 (#2188)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.9.1 (#2192)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

gabenddos committed:
fix: broken links by adding default append slashes to internal redirects (#2186)

simonhyll committed:
fix: i18n tracker create page link

simonhyll committed:
fix: i18n pages structure (#2193)

vasfvitor committed:
fix: i18n enable pages (#2194)

simonhyll committed:
fix: landing page styling for all languages

simonhyll committed:
fix: links to start page for i18n

simonhyll committed:
fix: beta.tauri.app to v2.tauri.app

simonhyll committed:
fix: next branch to v2

simonhyll committed:
fix: merge application debug with overview and improve debug only docs (#2176)

Co-authored-by: Vitor Ayres <gitme@virtuaires.com.br>

simonhyll committed:
feat: adding format checker for PRs

simonhyll committed:
fix: format requires version 20

simonhyll committed:
feat: adding pre-commit and formatting stuff (#2196)

simonhyll committed:
[ci] format

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.1.27 (#2199)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: tabs are a sin

renovate[bot] committed:
fix(deps): update dependency prettier-plugin-astro to ^0.14.0 (#2201)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
feat: d2 diagrams, process model (#2202)

simonhyll committed:
[ci] format

simonhyll committed:
feat: added diagram for Tauri Architecture (#2204)

dreyfus92 committed:
[ci] format

renovate[bot] committed:
fix(deps): update dependency starlight-links-validator to ^0.9.0 (#2205)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: general frontend configuration improvements

simonhyll committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

simonhyll committed:
fix: added descriptions to link cars in upgrade and migrate

simonhyll committed:
fix: core concepts overview page no longer wip

simonhyll committed:
fix: switch to lorenzo's sidebar

simonhyll committed:
[ci] format

simonhyll committed:
fix: create-a-project no longer wip

simonhyll committed:
fix: minor edits

simonhyll committed:
[ci] format

simonhyll committed:
fix: lp styling

simonhyll committed:
[ci] format

stretch07 committed:
Add environment variables documentation (#2200)

Co-authored-by: vasfvitor <gitkey@virtuaires.com.br>

renovate[bot] committed:
fix(deps): update dependency jsdom to v24.1.0 (#2208)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: formatting fix

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-markdown to v4.0.3 (#2210)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.9.2 (#2209)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

vasfvitor committed:
feat: bun command tab and lower case commands (#2212)

simonhyll committed:
feat: added ipc diagrams (#2203)

simonhyll committed:
[ci] format

renovate[bot] committed:
fix(deps): update dependency shiki to v1.6.1 (#2214)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

kareemmahlees committed:
fix(links): upgrade & migrate cards' links (#2213)

GuoJikun committed:
i18(zh-cn): Keep the `sidebar` consistent with the English version of the document and add Chinese translation. (#2198)

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to v0.23.2 (#2215)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.12.13 (#2217)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: renovate update git submodules

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to e0885cc (#2218)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: match pnpm version with plugins-workspace

renovate[bot] committed:
chore(deps): update packages/tauri digest to 0a6e62a (#2220)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: netlify doesn't read engines version

simonhyll committed:
fix: disable strict package manager checking for the formatters sake

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 5661cce (#2219)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update pnpm to v9.1.3 (#2221)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

vanvuongngo committed:
Guide(Frontend): Add Qwik (#1798)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>

simonhyll committed:
[ci] format

tauri-bot committed:
[ci] format

simonhyll committed:
fix: qwik guide cleanup

simonhyll committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

simonhyll committed:
fix: formatting workflow ruined the file

simonhyll committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

tauri-bot committed:
[ci] format

simonhyll committed:
fix: formatting workflow on PR instead of v2 and no push

simonhyll committed:
fix: qwik formatting

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to 20ceb6d (#2222)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: schedule git-submodules update to weekly

renovate[bot] committed:
chore(deps): update packages/tauri digest to 916a26c (#2223)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: incorrect schedule format apparently

renovate[bot] committed:
fix(deps): update dependency sass to v1.77.3 (#2224)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: remove duplicate workflow

simonhyll committed:
fix: point to v1.tauri.app instead of tauri.app

renovate[bot] committed:
chore(deps): update pnpm to v9.1.4 (#2227)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

tillmann-crabnebula committed:
Security Reference and Documentation Polishing (#2216)

Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>

simonhyll committed:
fix: add hover effect for title links and remove unnecessary css

simonhyll committed:
fix: broken links

simonhyll committed:
fix: remove (CLI) as it's redundant

simonhyll committed:
fix: bump docs.rs link from 19 to 22

renovate[bot] committed:
fix(deps): update dependency sass to v1.77.4 (#2229)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
i18(zh-cn): translate `qwik.mdx` and `sveltekit.mdx` (#2226)

Co-authored-by: DK Liao <dklassic@gmail.com>

simonhyll committed:
fix: add libxdo-dev to debian dependencies

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.13.0 (#2232)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 0f739db (#2233)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.13.0 (#2235)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 3cca5c2 (#2234)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency prettier to v3.3.0 (#2236)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.1.28 (#2237)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.6.2 (#2238)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.0 (#2239)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: wrong github link

Accidentally set it before to `/release/` instead of `/releases/` during the restructure.

simonhyll committed:
fix: missing trailing slash for internal links

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.1 (#2244)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
i18(zh-cn): Synchronizing content (#2231)

Co-authored-by: DK Liao <dklassic@gmail.com>

Embers-of-the-Fire commented on src/content/docs/zh-cn/features/dialog.mdx:
```diff
@@ -0,0 +1,299 @@
+安装对话框插件以开始使用。
+
+<Tabs>
+    <TabItem label="Automatic">
```
Missing translation.

Embers-of-the-Fire commented on src/content/docs/zh-cn/features/dialog.mdx:
````diff
@@ -0,0 +1,299 @@
+    }
+    ```
+    </TabItem>
+    <TabItem label="Manual">
````
Missing translation.

Embers-of-the-Fire commented on src/content/docs/zh-cn/guides/create/index.mdx:
```diff
@@ -0,0 +1,77 @@
+按照提示选择您的项目名称、前端语言、包管理器和前端框架，以及前端框架选项（如果适用）。
+:::tip[不确定该选择什么？]
+
+我们建议从基础的模版开始 (无框架的的HTML、CSS 和 JavaScript)。你随时可以 [集成一个前端框架](/guides/create/existing-project)。
```
Missing whitespace.

Embers-of-the-Fire commented on src/content/docs/zh-cn/guides/create/index.mdx:
```diff
@@ -0,0 +1,77 @@
+
+## 使用 `create-tauri-app`
+
+要开始使用 `create-tauri-app`的话，请在要设置项目的文件夹中运行以下命令之一。 如果您不确定使用哪个命令，我们建议在Linux和macOS上使用Bash命令，在Windows上使用PowerShell命令。
```
Missing whitespace.

Embers-of-the-Fire commented on src/content/docs/zh-cn/guides/create/index.mdx:
```diff
@@ -0,0 +1,77 @@
+- 使用 `create-tauri-app` 创建一个新APP
+- 使用Tauri CLI手动创建应用程序
+- 将移动设备添加到现有项目 (修改这里的内容 [integrate mobile](./mobile))
+- 特定于前端框架的指南（现在仅限于React、Angular、Vue、Svelte和Solid）
```
Missing whitespace.

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.2 (#2248)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency prettier to v3.3.1 (#2250)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.9.3 (#2249)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to v0.23.4 (#2246)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro-d2 to ^0.3.0 (#2245)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update pnpm to v9.2.0 (#2251)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

FabianLars committed:
docs(start): Fix nuxt config

EthnTuttle committed:
docs: Fix typo in Develop/plugin (#2255)

renovate[bot] committed:
fix(deps): update dependency astro to v4.10.0 (#2257)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
feat: turns theme selector into a button instead of a dropdown (#2256)

murnifine committed:
Update nextjs.mdx (#2254)

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to ^0.24.0 (#2253)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @lorenzo_lewis/starlight-utils to v0.1.1 (#2252)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
feat: added a splashscreen lab (#2106)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.6.3 (#2258)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 264a044 (#2261)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.10.1 (#2263)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 6ada2af (#2264)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to 61e8daa (#2260)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 5b76994 (#2262)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

zennolux committed:
fix(prerequisites): remove duplicated env path of `NDK_HOME` (#2265)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.1.29 (#2266)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to v0.24.1 (#2268)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update pnpm to v9.3.0 (#2270)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Levminer committed:
docs: Update tray migration guide to new clicktype (#2267)

renovate[bot] committed:
chore(deps): update dependency simple-git to v3.25.0 (#2269)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency prettier to v3.3.2 (#2272)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

tillmann-crabnebula committed:
Add Initial Security Learning Tutorial (#2241)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

cuervolu committed:
i18(es): add spanish translation to the Upgrade from Tauri 1.0 guide. (#2195)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>
Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency astro to v4.10.2 (#2274)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.6.4 (#2275)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

RoseBlume committed:
docs: Documentation for distributing with the Arch User Repository. (#2146)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

GuoJikun committed:
i18(zh-cn): Synchronizing content (#2271)

Co-authored-by: DK Liao <dklassic@gmail.com>

dreyfus92 committed:
chore: update PR template (#2276)

joshtenorio committed:
docs: Migrate Embedding Additional Files from v1 (#2206)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

robertosw committed:
docs(draft): Instructions for creating a flatpack (#1760)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>
Co-authored-by: FabianLars <fabianlars@fabianlars.de>

0PandaDEV committed:
docs: added updater plugin guide (#2207)

Co-authored-by: Vitor Ayres <gitkey@virtuaires.com.br>
Co-authored-by: Vitor Ayres <gitme@virtuaires.com.br>
Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency sass to v1.77.5 (#2280)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

FabianLars committed:
chore: add updater redirect

RoseBlume committed:
docs: Add Flatpak Docs (#2279)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

FabianLars committed:
docs: Clean up wip pages (#2282)

Co-authored-by: Tillmann <112912081+tillmann-crabnebula@users.noreply.github.com>
Co-authored-by: Chip Reed <chip@crabnebula.dev>

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to v0.24.2 (#2285)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

vdang-crabnebula committed:
docs: guide to capabilities in multi-windows app (#2283)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

FabianLars committed:
docs: Port CrabNebula Cloud intro

FabianLars committed:
docs: Remove incompat section from brownfield page

Rigidity committed:
fix: remove old incompatibilities link (#2289)

renovate[bot] committed:
fix(deps): update dependency shiki to v1.6.5 (#2296)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.1.30 (#2297)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to 8ccd423 (#2293)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to a8310f4 (#2294)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

RoseBlume committed:
Add Distribute Overview (#2301)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>
Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

FabianLars committed:
ci: Change workflow to check formatting

renovate[bot] committed:
fix(deps): update dependency shiki to v1.7.0 (#2302)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency starlight-blog to ^0.9.0 (#2303)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: FabianLars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency astro to v4.10.3 (#2305)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.3 (#2307)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency sass to v1.77.6 (#2308)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.5 (#2309)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.5 (#2310)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to v0.24.3 (#2312)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.6 (#2315)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

derekblank committed:
Update RSS LinkCard descriptions (#2314)

renovate[bot] committed:
chore(deps): update pnpm to v9.4.0 (#2306)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to 5e6ea7c (#2299)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 17c42fd (#2300)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

kiawin committed:
docs: Add bun command for plugin docs (#2298)

Co-authored-by: Sian Lerk Lau <kiawin@users.noreply.github.com>

simonhyll committed:
fix: formatting

simonhyll committed:
fix: i18n auto labels

simonhyll committed:
fix: renovate auto label pr's

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.0 (#2318)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to v0.24.4 (#2319)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.7 (#2321)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typescript to v5.5.2 (#2320)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
i18(zh-cn): Translate aur and flatpak under the distribute directory (#2317)

Co-authored-by: DK Liao <dklassic@gmail.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 58821fc (#2295)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Simon Hyll <hyllsimon@gmail.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.8.0 (#2322)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.9.0 (#2323)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to ce9a97c (#2324)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

EthnTuttle committed:
fix: typo - index.mdx (#2329)

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.1 (#2328)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.1 (#2331)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.8 (#2326)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update astro monorepo (#2332)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.9.1 (#2335)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.9 (#2336)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.2 (#2338)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

FabianLars committed:
docs(sql): Fix BaseDir reference.

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.3 (#2339)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.10.0 (#2342)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @astrojs/starlight to v0.24.5 (#2344)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency starlight-blog to v0.9.1 (#2350)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.2 (#2351)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

jbolda committed:
Tauri Board Elections 2024 (#2345)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to e2671d7 (#2346)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 6dedb3f (#2347)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Vitor Ayres <gitkey@virtuaires.com.br>

jbolda committed:
Blog Post Sidebar Sorts By Date (#2349)

Co-authored-by: vasfvitor <gitkey@virtuaires.com.br>

RoseBlume committed:
docs: Add Snapcraft docs (#2290)

Co-authored-by: FabianLars <fabianlars@fabianlars.de>

RoseBlume committed:
docs: update Distribute overview (#2304)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>
Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 911242f (#2348)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Sir-Thom committed:
docs: Adding RPM doc (#2316)

renovate[bot] committed:
fix(deps): update dependency typescript to v5.5.3 (#2353)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
feat: edit page links open vscode during development

simonhyll committed:
fix: article ordering in learn category

lucasfernog committed:
blog: announce Tauri 1.7 (#2354)

simonhyll committed:
fix: use intended expressiveCode style override

simonhyll committed:
fix: netlify developer frame csp

renovate[bot] committed:
fix(deps): update dependency starlight-links-validator to v0.9.1 (#2358)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.10.1 (#2357)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency starlight-blog to ^0.10.0 (#2359)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.4 (#2360)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Nabellaleen committed:
fix: fix cargo command in create-project.mdx (#2361)

simonhyll committed:
Revert "fix(deps): update dependency astro to v4.11.4" (#2365)

FabianLars committed:
ci: update workflow versions

FabianLars committed:
ci: Remove pnpm version from workflow

RoseBlume committed:
docs: Add Github Actions Docs (#2313)

Co-authored-by: Breno A <git@breno.tech>
Co-authored-by: FabianLars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency starlight-blog to ^0.11.0 (#2368)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.5 (#2362)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

zrq8 committed:
docs: fix typo on the package name (#2369)

FabianLars committed:
docs: fix nextjs frontendDist

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.10 (#2371)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.3 (#2375)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 24aaf0f (#2374)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.10.2 (#2376)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.10.3 (#2378)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

RomainMazB committed:
docs: Fix environment variables link (#2380)

charlesschaefer committed:
docs: Adding an example of how to trigger a notification from rust code. (#2381)

Co-authored-by: FabianLars <fabianlars@fabianlars.de>

AurimarL committed:
fix nextjs beforeBuildCommand (#2383)

renovate[bot] committed:
fix(deps): update dependency @lorenzo_lewis/starlight-utils to v0.1.2 (#2382)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
i18(zh-cn): Synchronizing content (#2352)

renovate[bot] committed:
chore(deps): update pnpm to v9.5.0 (#2379)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to ac9a25c (#2373)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to e2671d7 (#2372)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

simonhyll committed:
fix: trailing slashes always on

simonhyll committed:
fix: disable trailing slash enforcement

amrbashir committed:
feat: android signing guide (#2337)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>

jbolda committed:
Add Description To Store Plugin (#2384)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>

simonhyll committed:
Merge branch 'v2' into next
